docs/application/dotnet/guides/tizen.circularUI/ConfirmationPopup.md

```diff
@@ -0,0 +1,84 @@
+summary: ConfirmationPopup control guide
+---
+
+# ConfirmationPopup
+`ConfirmationPopup` represents The Tizen Wearable EFL two button popup style. The `ConfirmationPopup` is used to select one of the two options such as cancel or done, and share or delete.
+
+*Since `ConfirmationPopup` does not adhere as a control, this popup cannot be set in the XAML file.*
+
+![confirmation_popup1](data/confirmation_popup1.png)
+
+## Create ConfirmationPopup
+`ConfirmationPopup.Title` property sets the title of the popup. You can set the `Content` property with Layouts such as `StackLayout` or `ScrollView`.
+`ConfirmationPopup.FirstButton` property sets left side button. `ConfirmationPopup.SecondButton`property sets right side button. You can set` FirstButton` and `SecondButton` using `MenuItem`. You should add code at `Command` or `Clicked` event handler for controlling `MenuItem` clicked.
+
+_The code example of this guide uses WearableUIGallery's TCConfirmationPopup code at the test\WearableUIGallery\WearableUIGallery\TC\TCConfirmationPopup.xaml.cs_
+
+For more information, see [ConfirmationPopup  API reference](https://samsung.github.io/Tizen.CircularUI/api/Tizen.Wearable.CircularUI.Forms.ConfirmationPopup.html).
+
+**C# file**
+```cs
+            var leftButton = new MenuItem()
+            {
+                Icon = new FileImageSource
+                {
+                    File = "image/b_option_list_icon_share.png",
+                },
+                Command = new Command(() =>
+                {
+                    /* must insert code when user press left button */
+                })
+            };
+
+            ...
+
+            _popUp1 = new ConfirmationPopup();
+            _popUp1.FirstButton = leftButton;
+            _popUp1.SecondButton = rightButton;
+            _popUp1.Title = "Popup title";
+            _popUp1.Content = new StackLayout()
+            {
+                HorizontalOptions = LayoutOptions.FillAndExpand,
+                Children =
+                {
+                    new Label
+                    {
+                        Text = "Will be saved, and sound, only on the Gear.",
+                    },
+                    new StackLayout
+                    {
+                        Orientation = StackOrientation.Horizontal,
+                        Padding = new Thickness(0, 30, 0, 30),
+                        Children =
+                        {
+                            checkbox,
+                            new Label
+                            {
+                                Text = "Do not repeat",
+                            }
+                        }
+                    }
+                }
+            };
+    }
+```
+
+## How to Set Long Text in ConfirmationPopup
+ConfirmationPopup has `Text` property. This property is useful to set long text.
+If a long text was set by `Text` property, you don't need to add `StackLayout`, `ScrollView` and `Label` for displaying long text.
```

Comment:
StackLayout`, `ScrollView` and `Label` for displaying long text.
Please add serial comma:
StackLayout`, `ScrollView` and, `Label` for displaying long text.